File: app/src/main/java/com/rorpage/purtyweather/models/BaseWeatherInfoUnit.java
package com.rorpage.purtyweather.models;

import java.util.List;

public class BaseWeatherInfoUnit {
    public long dt;
    public long sunrise;
    public long sunset;
    public int pressure;
    public int humidity;
    public double dew_point;
    public double uvi;
    public int clouds;
    public int visibility;
    public double wind_speed;
    public int wind_deg;

    public List<Weather> weather;
}

File: app/src/main/java/com/rorpage/purtyweather/models/BaseWeatherInfoUnit.kt
package com.rorpage.purtyweather.models

import com.google.gson.annotations.SerializedName

open class BaseWeatherInfoUnit {
    var dt: Long = 0
    var sunrise: Long = 0
    var sunset: Long = 0
    var pressure = 0
    var humidity = 0
    @SerializedName("dew_point")
    var dewPoint = 0.0
    var uvi = 0.0
    var clouds = 0
    var visibility = 0
    @SerializedName("wind_speed")
    var windSpeed = 0.0
    @SerializedName("wind_deg")
    var windDeg = 0
    var weather: List<Weather>? = null
}